modules/microshift-cni-customization-matrix.adoc

```diff
@@ -0,0 +1,46 @@
+
+:_mod-docs-content-type: REFERENCE
+[id="microshift-nw-ovn-kubernetes-matrix_{context}"]
+= {microshift-short} CNI feature matrix
+
+The following table summarizes the status of OVN-K networking features that are either present as defaults, supported for configuration, or excluded from the {microshift-short} service:
+
+.Default {microshift-short} OVN-K network plugin features and customization status
```

Comment:
Not all the network features in MicroShift are provided by OVN-K. https://docs.openshift.com/container-platform/4.14/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html#nw-ovn-kubernetes-matrix_about-ovn-kubernetes